homepage.css

```diff
@@ -0,0 +1,4 @@
+*{
```

Comment:
Why do we need to make padding and margin zero for everything?

It will affect every tag on the page.
So, remove this and use a different approach.

Styles for one class(tag, etc) should not affect others.

Reply:
okk i will do this now

Reply:
This is still not fixed!